node-notes-cli/app.js

```diff
@@ -0,0 +1,49 @@
+const fs = require('fs');
+const parsed = require('./data.json');
+
+// READ
+if (process.argv[2] === 'read') {
+  fs.readFile('./data.json', 'utf8', (err, data) => {
+    if (err) {
```

Comment:
Since you used `require()` to load the data model, you can remove this `readFile` call.